ISISScientificComputing/autoreduce
# Ansible Docker install

### Summary of work
I have added a Docker installation to the ansible-compute scripts. 

### How to test your work
Install Autoreduction on a fresh VM using the ansible-compute scripts, then check that Docker has been installed successfully.
